txbuilder/txbuilder.go

```diff
@@ -279,6 +280,12 @@ func SetValidRangeTo(to int64) Option {
 	}
 }
 
+func SetJsonMetadata(path string) Option {
```

Comment:
func SetMetadata(json string)

Reply:
should be detailed naming here because having 2 ways to attach metadata (json & cbor). Maybe `cbor` will be used later

Reply:
then `SetJSONMetadata`